app/components/post_list/thread_overview/thread_overview.tsx

```diff
@@ -101,35 +99,8 @@ const ThreadOverview = ({isSaved, repliesCount, rootId, rootPost, style, testID}
 
     const saveButtonTestId = isSaved ? `${testID}.unsave.button` : `${testID}.save.button`;
 
-    let repliesCountElement;
-    if (repliesCount > 0) {
-        repliesCountElement = (
-            <FormattedText
-                style={styles.repliesCount}
-                id='thread.repliesCount'
-                defaultMessage='{repliesCount, number} {repliesCount, plural, one {reply} other {replies}}'
-                testID={`${testID}.replies_count`}
-                values={{repliesCount}}
-            />
-        );
-    } else if (isFetchingThread) {
```

Comment:
As discussed, proposed removing the `Loading replies...` state: we don't want a flash of this component if we don't yet know there aren't replies.